core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts

```diff
@@ -49,6 +49,10 @@ export class DotExperimentsConfigurationSchedulingAddComponent implements OnInit
     today = new Date();
     initialDate = new Date();
     minEndDate: Date;
+    maxEndDate: Date;
+
+    private time14days = 12096e5;
```

Comment:
Add this to the constant file of dotExperiments.

Reply:
done